modules/desktop/graphics/ewwbar.nix

```diff
@@ -166,8 +167,10 @@ let
             ${eww} open --no-daemonize --screen "$display" bar --id bar:$display --arg screen="$display"
         done
 
+      ${lib.optionalString useGivc ''
         # Open a widget for hotkey indicators only on main display
         #${eww} open --no-daemonize hotkey-indicator
+      ''}
```

Comment:
As this was apparently left behind by me :hand: as a commented out line, we could just remove it entirely.
Otherwise it doesn't affect functionality.